keyboards/sck/m0116b/keymaps/default/keymap.c

```diff
@@ -0,0 +1,126 @@
+		KC_LSFT,          KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+		KC_LCAP, KC_LALT, KC_LGUI, KC_GRV,                   KC_SPC,             LT(1, KC_BSLS), KC_LEFT, KC_UP,   KC_DOWN, KC_RGHT, KC_P0,            KC_PDOT
+),
+/* Keymap Layer 1: (Layer 1) Layer 1
+   * ,---------------------------------------------------------------------------------.
+   * |                     |RESET|                                                     |
+   * `---------------------------------------------------------------------------------'
+   * ,-----------------------------------------------------------. .-------------------.
+   * |   |F1 |F2 |F3 |F4 |F5 |F6 |F7 |F8 |F9 |F10|F11|F12|       | |    |    |    |    |
+   * |-----------------------------------------------------------| |-------------------|
+   * |     |   |   |   |   |   |   |   |   |   |   |   |   |     | |    |    |    |    |
+   * |-----------------------------------------------------|     | |-------------------|
+   * |       |   |   |   |   |   |   |   |   |   |   |   |       | |    |    |    |    |
+   * |-----------------------------------------------------------' |-------------------|
+   * |        |   |   |   |   |   |   |   |   |   |   |          | |    |    |    |    |
+   * |-----------------------------------------------------------' `--------------|    |
+   * |    |    |    |   |    M0116B       |   |   |    |    |    | |         |    |    |
+   * `-----------------------------------------------------------' `-------------------'
+   */
+[1] = LAYOUT_M0116( /* Layer 1 */
+	                                                          RESET,
+		KC_TRNS, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   SAFE_RANGE,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS
+    ),
+};
+
+//const uint16_t PROGMEM fn_actions[] = {
+//
+//};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
+{
+  // MACRODOWN only works in this function
+      switch(id) {
+        case 0:
+          if (record->event.pressed) {
+            register_code(KC_RSFT);
+          } else {
+            unregister_code(KC_RSFT);
+          }
+        break;
+      }
+    return MACRO_NONE;
+};
+
+
```

Comment:
Lines 71-90 should be deleted. These functions are deprecated, and the keymap isn't using them anyway.

Reply:
taken care of